playwright/tests/common/inAppGuide.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from "@playwright/test";
+
+import { test } from "../../fixtures";
+
+test.describe("In-app guide", () => {
+  test.run({ onboarded: false })(
+    "I can see the in-app guide open by default",
+    async ({ sliceMachinePage }) => {
+      await sliceMachinePage.gotoDefaultPage();
+
+      await expect(sliceMachinePage.inAppGuideDialog.title).toBeVisible();
```

Comment:
Follow-up from a previous comment about evaluating element queries lazily: this test would look something like this when changing from an instance property to a getter:

```ts
    async ({ sliceMachinePage }) => {
      await sliceMachinePage.gotoDefaultPage();

      await expect(sliceMachinePage.inAppGuideDialog.getTitle()).toBeVisible();
    },
  );
```

Reply:
No need, Locator are kind of already a function. Wrapping it in a function yourself will not make any difference. It's evaluated only when you await them.